bugbug/bug_features.py

```diff
@@ -21,28 +21,36 @@ def field(bug, field):
     return None
 
 
-class has_str(object):
+class single_bug_feature(object):
+    pass
+
+
+class tuple_bug_feature(object):
```

Comment:
Nit: replace `tuple` with `couple` everywhere, since tuple can have many elements in general